Source/WebCore/css/parser/CSSTokenizer.cpp

```diff
@@ -140,16 +140,26 @@ unsigned CSSTokenizer::tokenCount()
     return m_tokens.size();
 }
 
-static bool isNewLine(UChar cc)
+bool CSSTokenizer::isWhitespace(CSSParserTokenType type)
+{
+    return type == NonNewlineWhitespaceToken || type == NewlineToken;
+}
+
+bool CSSTokenizer::isNewline(UChar cc)
 {
     // We check \r and \f here, since we have no preprocessing stage
     return (cc == '\r' || cc == '\n' || cc == '\f');
 }
 
+CSSParserToken CSSTokenizer::newLine(UChar /*cc*/)
```

Comment:
The commented out `cc` doesn’t add anything. Please just omit it. But also why is this function named “new line”, two words?

Reply:
Yup, but I noticed other similar codepoint functions in the code had the commented out `cc` so I added it for clarity, but I agree it's unnecessary. Removed.

eg: 
```
CSSParserToken CSSTokenizer::colon(UChar /*cc*/)
{
    return CSSParserToken(ColonToken);
}

CSSParserToken CSSTokenizer::semiColon(UChar /*cc*/)
{
    return CSSParserToken(SemicolonToken);
}
```

Reply:
Yes, this is likely remnants of a difference in coding style from the Chromium contributors since this tokenizer was originally done there and imported back to WebKit.